.ocamlformat

```diff
@@ -1,4 +1,3 @@
-version=0.21.0
```

Comment:
rebasing should make it use 0.24.1

Reply:
Oops, this change was not meant, I removed it from this branch. Thanks.